persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraPersistentEntityRegistry.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence.cassandra
+
+import java.util.{ Optional, UUID }
+
+import akka.NotUsed
+import akka.japi.Pair
+import akka.stream.javadsl
+import com.lightbend.lagom.javadsl.persistence.{ AggregateEvent, AggregateEventTag, PersistentEntityRegistry }
+
+/**
+ * Functionality specific to the Cassandra persistent entity registry.
+ */
+trait CassandraPersistentEntityRegistry extends PersistentEntityRegistry {
+
+  /**
+   * A stream of the persistent events that have the given `aggregateTag`, e.g.
+   * all persistent events of all `Order` entities.
+   */
+  def uuidEventStream[Event <: AggregateEvent[Event]](
+    fromOffset:   Optional[UUID],
```

Comment:
If we can convert between UUID and Long, wouldn't it make more sense to get rid of `Optional` in the signature? That means three overloaded methods, but it seems to me the API would benefit. (Also, this would be a binary compatible change)